src/RadhydroSimulation.hpp

```diff
@@ -919,13 +924,15 @@ auto RadhydroSimulation<problem_t>::advanceHydroAtLevel(amrex::MultiFab &state_o
 		}
 
 		// prevent vacuum
-		HydroSystem<problem_t>::EnforceDensityFloor(densityFloor_, stateFinal);
+		HydroSystem<problem_t>::EnforceLimits(densityFloor_, pressureFloor_, speedCeiling_, tempCeiling_, tempFloor_, stateFinal);
 
 		if (useDualEnergy_ == 1) {
 			// sync internal energy (requires positive density)
 			HydroSystem<problem_t>::SyncDualEnergy(stateFinal);
 		}
 
+		HydroSystem<problem_t>::EnforceLimits(densityFloor_, pressureFloor_, speedCeiling_, tempCeiling_, tempFloor_, stateFinal);
```

Comment:
With the current design that limits the temperature for both `Etot - Ekin` and `Eint` separately, I don't think it should be necessary to call `EnforceLimits` twice.

Reply:
Is there a way to make this change without having to push stuff all over again?

Reply:
No. What is the issue with pushing again? It takes some time for the tests to run - is that the issue?